metrics/metric.py

```
@@ -0,0 +1,887 @@
+
+            try:
+                if 'data' not in r.json():
+                    # logger.info(f'Fetching {l} data from SemanticScholar.')
+                    return citation_count, influentionCC
+                    # raise ConnectionError
+
+                for item in r.json()['data']:
+                    if int(item['citationCount']) >= 0:
+                        citation_count.append(int(item['citationCount']))
+                        influentionCC.append(int(item['influentialCitationCount']))
+                        l += 1
```

Ruff (0.8.2)

Ambiguous variable name: `l`

(E741)